src/components/nodes/association/association.vue

```diff
@@ -17,48 +19,55 @@ export default {
       shape: null,
       definition: null,
       sourceShape: null,
-      validConnections: {
-        'processmaker-modeler-text-annotation': [
-          'processmaker-modeler-task',
-          'processmaker-modeler-start-event',
-          'processmaker-modeler-end-event',
-          'processmaker-modeler-exclusive-gateway',
-          'processmaker-modeler-inclusive-gateway',
-          'processmaker-modeler-parallel-gateway',
-        ],
-      },
+      target: null,
+      anchorPadding: 25,
+      laneShape: LaneShape.id,
```

Comment:
There's no need to add this to the component data. The name is also misleading as it's not referring to the jointjs shape/model but a string ID.

Import it above as:
```
import { id as laneId } from '../poolLane';
```

And reference it directly as `laneId` in the component below.